app/controllers/posts_controller.rb

```diff
@@ -31,6 +31,23 @@ def create
     end
   end
 
```

Comment:
- [ ] Kindly, make sure that you implemented the authorization for Posts using `cancancan` in your PostsController Class by adding `load_and_authorize_resource` which loads up the necessary model instance for CRUD operations. have a look on the snippet code below:
```ruby
  class PostsController < ApplicationController
    load_and_authorize_resource

    ... code here ...
  end
```

Reply:
Hello @RolandM99 can you please specify the authorization you are suggesting, because on this end all authorization is implemented.

User can't delete the post or comment which is not his.

Reply:
Let's Go on slack for more explanation